lib/rubocop/result_cache.rb

```diff
@@ -78,7 +79,7 @@ def self.cleanup(config_store, verbose, cache_root = nil)
 
     def self.cache_root(config_store)
       root = config_store.for('.')['AllCops']['CacheRootDirectory']
-      root = Dir.tmpdir if root == '/tmp'
+      root = File.join(Dir.tmpdir, Etc.getlogin) if root == '/tmp'
```

Comment:
I guess we would always want the user name in the path, also when `AllCops: CacheRootDirectory` has been set to another value. So add it on the next line instead. It could go after `'rubocop_cache'`.